Xdl.Internship.Core/Xdl.Internship.Core.DataAccess.MongoDB/CollectionProviders/CollectionProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using MongoDB.Driver;
+using Xdl.Internship.Core.DataAccess.MongoDB.ConnectionFactories;
+using Xdl.Internship.Core.Models.MongoDB;
+
+namespace Xdl.Internship.Core.DataAccess.MongoDB.CollectionProviders
+{
+    public class DefaultCollectionProvider : ICollectionProvider
```

Comment:
pls rename file to DefaultCollectionProvider

Reply:
resolved